src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/SolutionExplorer/NuGetWorkspaceCommandHandler.cs

```diff
@@ -86,8 +86,7 @@ public bool QueryStatus(List<WorkspaceVisualNodeBase> selection, Guid pguidCmdGr
                             if (IsSolutionOnlySelection(selection))
                             {
                                 var isRestoreActionInProgress = _restoreCommandHandler.IsRestoreActionInProgress();
-                                cmdf = (uint)((isRestoreActionInProgress ? 0 : OLECMDF.OLECMDF_ENABLED)
-                                    | OLECMDF.OLECMDF_SUPPORTED);
+                                cmdf = (uint)((isRestoreActionInProgress ? 0 : OLECMDF.OLECMDF_ENABLED) | OLECMDF.OLECMDF_SUPPORTED);
```

Comment:
There's no functional change here is there? 

I'd revert it if that's the case.

Reply:
Like the above, I didn't see this comment in time. If you feel strongly about it, I'm happy to push a new change to roll this back.

Reply:
That'd defeat the purpose, because this looks like a formatting change, so the next time anyone does a change in that file, it'll likely do it automatically anyways. 

Don't worry about it :) 
Likely that this file will undergo more changes soon enough.